etc/profile-a-l/librewolf.profile

```diff
@@ -19,6 +19,8 @@ whitelist ${HOME}/.librewolf
 #noblacklist ${HOME}/.mozilla
 #whitelist ${HOME}/.mozilla
 
+include whitelist-usr-share-common.inc
```

Comment:
That's firefox' block, should we add some of them too?
https://github.com/netblue30/firejail/blob/ba2d8a8e9bebd274fa5aa554e792164a7981e110/etc/profile-a-l/firefox.profile#L22-L28

Reply:
no it's not needed

Reply:
It's not needed to start librewolf. But what is with other features. Does it support `/usr/share/webext`? Or do we want `/usr/share/doc`? (The questions are for anybody who has a opinion).

Reply:
I think we should add the relevant ones. AFAICT librewolf supports webext. So the only path NOT to whitelist would be /usr/share/firefox IMO.